src/hook/api/tak/useFetchZipFile.ts

```diff
@@ -27,7 +27,20 @@ async function fetchZipFile(
   }
 
   const data = (await res.json()) as FilesResponse;
-  const fileIndex = os === "iOS" ? 1 : 0; // iTAK package is at index 1
+
+  // Map OS to file indices: Other, iOS and tracker
+  const osToFileIndex: { [key: string]: number } = {
+    Other: 0, // ATAK package
+    iOS: 1, // iTAK package
+    tracker: 2, // Tracker package
+  };
+
+  const fileIndex = osToFileIndex[os];
```

Comment:
This could be using a helper function which uses a `switch case` for safer implementation.

Although since the input value comes from the pre-defined options it doesn't matter that much.